lib/chef/web/core/helpers.rb

```diff
@@ -0,0 +1,34 @@
+require 'chef/web/core'
+require 'chef/web/core/component'
+require 'nokogiri'
+
+class Chef
+  module Web
+    module Core
+      module Helpers
+
+        def chef_logo(element=:div, opts={})
```

Comment:
Since the helper's just returning a String here, I'm not sure anything can be done in web-core to prevent the need to call `html_safe` (or the like) on these helpers in Rails.